### ISSUE TITLE
Usher FAB-15824 patchset through

### ISSUE THREAD

**Issue by rpocase** [APGGroeiFabriek/PIVT#15]
It would be nice to see the [FAB-15824 patchset](https://gerrit.hyperledger.org/r/c/fabric/+/32197) ushered through. That being a key factor in the implementation makes tying any internal or community efforts based on these charts risky. There is the older release of these examples that don't depend on it, but I don't expect any patches for fixes based on that workflow.

Really just getting a note here to further highlight that this is still outstanding and a reminder to update the docs if/when the change is merged.

**Comment by rpocase** [comment]
@raftAtGit Any updates on this? Last I saw this was in a merge conflict state. 

I did see the extended image got bumped to 1.4.3 a couple weeks ago. As long as the patchset continues to get custom docker images there is a bit less risk, though still not ideal.

**Comment by raftAtGit** [comment]
No, no updates. It's in hands of Fabric guys, hoping it will be merged for 2.0 release.

extended image 1.4.3 was an experiment, it's actually based on Fabric 1.4.3. but not used by default in this repo yet. is required for `cryptogen 1.4.3` compatibility, which I will implement somewhere in near future.

**Comment by raftAtGit** [comment]
@rpocase btw, it can help if you write a comment on the patch set and also on the Jira ticket ;)

**Comment by rpocase** [comment]
It looks like fabric was finally [moved to github](https://github.com/hyperledger/fabric). The gerrit review said outstanding CR's should have been recreated as a PR, but I can't seem to find a matching PR for this. I expect the original patchset will need to be rebased from tip and reintroduced. I suspect it will naturally get more traction with the more intuitive nature of github's interface.

**Comment by raftAtGit** [comment]
only if integration tests can be completed in 120 minutes! they are cancelled after 120 minutes!

https://github.com/hyperledger/fabric/pull/322

**Comment by rpocase** [comment]
Looks like several open PRs are having the same issue. Their integration test suite seems a bit inconsistent in timing.

**Comment by raftAtGit** [comment]
yea, there is also a PR for making integration tests faster (by running in parallel) https://github.com/hyperledger/fabric/pull/312

**Comment by raftAtGit** [comment]
needed to close other one and start a new PL. all tests passed now though :)
https://github.com/hyperledger/fabric/pull/345
